reading/apis.livemd

```diff
@@ -147,7 +147,7 @@ You can make a simple cURL request by typing the following in your command line.
 curl https://www.example.com
 ```
 
-It's important to understand cURL because most APIs include documentation using cURL requests examples and we can use these simple curl requests to test an API or know how to send a request in our chosen Elixir API Client.
+It's important to understand cURL because most APIs include documentation using cURL request examples and we can use these simple curl requests to test an API or know how to send a request in our chosen Elixir API Client.
```

Comment:
```suggestion
It's important to understand cURL because most APIs include documentation using examples of cURL requests. We can use these simple cURL requests to test an API or know how to send a request using our chosen Elixir API Client.
```